client/my-sites/checkout/src/hooks/use-tos-foldable-card.tsx

```diff
@@ -7,12 +7,16 @@ import { getSelectedSiteId, getSectionName } from 'calypso/state/ui/selectors';
 
 export function useToSFoldableCard(): boolean {
 	const siteId = useSelector( ( state ) => getSelectedSiteId( state ) );
-
+	const section = useSelector( getSectionName );
 	const isJetpackNotAtomic = useSelector(
 		( state ) => isJetpackSite( state, siteId ) && ! isSiteAutomatedTransfer( state, siteId )
 	);
+
+	const enabledSections = [ 'business-plan-upgrade-upsell', 'checkout' ];
+
 	const isWPcomCheckout =
-		useSelector( getSectionName ) === 'checkout' &&
+		section != null &&
```

Comment:
Not sure if this is the best way to check for a null section. The goal is to only allow this hook to be opt in for different sections. When we remove this hook we can decide how to handle pushing this functionality across the board (but that also depends on how Jetpack and Akismet want to handle their terms of service?)